app/services/github_issues_author.rb

```diff
@@ -0,0 +1,55 @@
+class GithubIssuesAuthor
```

Comment:
This is my bread and butter of my integration with github.
This service is my only "source of truth" when it comes to the integration it does one job which is apply the action sent from the issues API.

currently it supports opening, editing, deleting an issue with the condition that it has the right payload to be processed.